lib/plugins/encrypted_parameters.rb

```diff
@@ -102,9 +102,11 @@ def find_or_create_kms_key
               Moonshot.config.parameters[@kms_key_parameter_name].set(key_arn)
               s.success "Created a new KMS Key for #{@kms_key_parameter_name.blue}!"
             else
-              key_arn = KmsKey.new(Moonshot.config.parameters[@kms_key_parameter_name].value).arn
+              kms=KmsKey.new(Moonshot.config.parameters[@kms_key_parameter_name].value)
+              key_arn = kms.arn
               s.success "Using existing KMS Key for #{@kms_key_parameter_name.blue}!"
             end
+            KmsKey.new(key_arn).update
```

Comment:
I think having tags included in creation makes one api call instead of two to aws. In the current approach its optimized to have single method. I suggest we make fewer api call as each call makes it take time and more exposed to api based errors.

Reply:
had to make stack_tags a class method for it to work, but did it.